comtypes/client/_events.py

```diff
@@ -275,8 +275,8 @@ def HandlerRoutine(dwCtrlType):
                                                                int(timeout * 1000),
                                                                len(handles), handles,
                                                                ctypes.byref(ctypes.c_ulong()))
-        except WindowsError, details:
-            if details.args[0] != RPC_S_CALLPENDING: # timeout expired
+        except WindowsError as details:
```

Comment:
If you change comma to `as` keyword, then you need to change it everywhere to keep consistency (not sure it's time for big re-factoring). `setup.py` script uses 2to3 fixer at the moment. So keeping comma wouldn't be a problem. With this small change I'd merge this PR without risk to have concerns from Enthought's colleagues. Thanks for understanding.